server/scsynth/scsynth_main.cpp

```diff
@@ -253,17 +253,13 @@ int main(int argc, char* argv[]) {
         case 'H':
             checkNumArgs(2);
             options.mInDeviceName = argv[j + 1];
-#ifdef __APPLE__
             if (i + 1 > argc || argv[j + 2][0] == '-') {
                 options.mOutDeviceName = options.mInDeviceName;
             } else {
                 // If there's a second argument then the user wants separate I/O devices
                 options.mOutDeviceName = argv[j + 2];
                 ++i;
             }
-#else
-            options.mOutDeviceName = options.mInDeviceName; // Non-Mac platforms always use same device
```

Comment:
what in this PR allows this to work right on Linux?

Reply:
There's no reason to arbitrarily make assumption about not being able to specify input and output devices independently. If you select a single device with the -H parameter, it will be used for both input and output.

I haven't tested ALSA to confirm options on linux, but I see no harm removing this limitation on that platform as well.

Reply:
cool, thanks for the explanation!